x-pack/legacy/plugins/ml/public/jobs/new_job_new/common/job_validator/util.ts

```diff
@@ -0,0 +1,161 @@
+          'must start and end with an alphanumeric character',
+      }
+    );
+    basicValidations.groupIds.message = msg;
+  } else if (validationResults.contains('job_group_id_already_exists')) {
+    basicValidations.groupIds.valid = false;
+    const msg = i18n.translate('xpack.ml.newJob.wizard.validateJob.groupNameAlreadyExists', {
+      defaultMessage:
+        'Group ID already exists. A group ID cannot be the same as an existing job or group.',
+    });
+    basicValidations.groupIds.message = msg;
+  }
+
+  if (validationResults.contains('model_memory_limit_units_invalid')) {
+    basicValidations.modelMemoryLimit.valid = false;
+    const str = `${ALLOWED_DATA_UNITS.slice(0, ALLOWED_DATA_UNITS.length - 1).join(', ')} or ${[
+      ...ALLOWED_DATA_UNITS,
+    ].pop()}`;
+    const msg = i18n.translate(
+      'xpack.ml.newJob.wizard.validateJob.modelMemoryLimitUnitsInvalidErrorMessage',
+      {
+        defaultMessage: 'Model memory limit data unit unrecognized. It must be {str}',
+        values: { str },
+      }
+    );
+    basicValidations.modelMemoryLimit.message = msg;
+  }
+
+  if (validationResults.contains('model_memory_limit_invalid')) {
+    basicValidations.modelMemoryLimit.valid = false;
+    const msg = i18n.translate(
+      'xpack.ml.newJob.wizard.validateJob.modelMemoryLimitRangeInvalidErrorMessage',
+      {
+        defaultMessage:
+          'Model memory limit cannot be higher than the maximum value of {maxModelMemoryLimit}',
+        values: { maxModelMemoryLimit: limits.max_model_memory_limit.toUpperCase() },
+      }
+    );
+    basicValidations.modelMemoryLimit.message = msg;
+  }
+
+  if (validationResults.contains('detectors_duplicates')) {
+    basicValidations.duplicateDetectors.valid = false;
+    const msg = i18n.translate(
+      'xpack.ml.newJob.wizard.validateJob.duplicatedDetectorsErrorMessage',
+      {
+        defaultMessage: 'Duplicate detectors were found.',
+      }
+    );
+    basicValidations.duplicateDetectors.message = msg;
+  }
+
+  if (validationResults.contains('bucket_span_empty')) {
+    basicValidations.bucketSpan.valid = false;
+    const msg = i18n.translate(
+      'xpack.ml.newJob.wizard.validateJob.bucketSpanMustBeSetErrorMessage',
+      {
+        defaultMessage: '{bucketSpan} must be set',
+        values: { bucketSpan: 'bucket_span' },
+      }
+    );
+
+    basicValidations.bucketSpan.message = msg;
+  } else if (validationResults.contains('bucket_span_invalid')) {
+    basicValidations.bucketSpan.valid = false;
+    const msg = i18n.translate(
+      'xpack.ml.newJob.wizard.validateJob.bucketSpanInvalidTimeIntervalFormatErrorMessage',
+      {
+        defaultMessage:
+          '{bucketSpan} is not a valid time interval format e.g. {tenMinutes}, {oneHour}. It also needs to be higher than zero.',
+        values: {
+          bucketSpan: jobConfig.analysis_config.bucket_span,
+          tenMinutes: '10m',
```

Comment:
These intervals don't need to be localized as ES aggregation intervals are not translated.

Reply:
Sorry, leaving them as `values` will ensure those intervals don't get translated. Ignore me :)